### PR TITLE
docs: fix a few things

### PR DESCRIPTION
**Pull request by jecluis**
Some things were either out of date or had minor mistakes.

This patch set fixes those.

Signed-off-by: Joao Eduardo Luis \<joao@suse.com>

### CODE REVIEW

**Review by votdev — DISMISSED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,7 +5,6 @@ organization.
 
 - Main: <https://github.com/aquarist-labs/s3gw>
 - Ceph: <https://github.com/aquarist-labs/ceph>
-- Tools: <https://github.com/aquarist-labs/s3gw-tools>
 - UI: <https://github.com/aquarist-labs/s3gw-ui>
 - Charts: <https://github.com/aquarist-labs/s3gw-charts>
```
Comment (m-ildefons) [inline CONTRIBUTING.md:11]: We might want to list the COSI repos here as well.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,13 +15,13 @@ We welcome new feature implementations, patches that fix bugs or code cleanup.
...
-git clone git@github.com:aquarist-labs/s3gw-tools
+git clone git@github.com:aquarist-labs/s3gw
 ```
 
 Or:
 
 ```bash
-git clone git://github.com/aquarist-labs/s3gw-tools
+git clone git://github.com/aquarist-labs/s3gw
 ```
```
Comment (m-ildefons) [inline CONTRIBUTING.md]: Stick with one of the two. This doesn't need to be an in-depth guide on how to use git.

File: docs/s3gw-repos.md [hunk 3]
```diff
@@ -3,7 +3,6 @@
 The S3 Gateway includes the following projects:
 
 - aquarist-labs/s3gw <https://github.com/aquarist-labs/s3gw>
-- aquarist-labs/s3gw-tools <https://github.com/aquarist-labs/s3gw-tools>
 - aquarist-labs/ceph <https://github.com/aquarist-labs/ceph>
 - aquarist-labs/s3gw-charts <https://github.com/aquarist-labs/s3gw-charts>
 - aquarist-labs/s3gw-status <https://github.com/aquarist-labs/s3gw-status>
```
Comment (m-ildefons) [inline docs/s3gw-repos.md:8]: Again, for completeness' sake I'd love to see the COSI repos listed here.

**Review by m-ildefons — CHANGES_REQUESTED** [review]

**Review by m-ildefons — APPROVED** [review]
LGTM

**Pull request merged**